js/simplemde.min.js

```diff
@@ -1,7 +1,7 @@
 /**
  * simplemde v1.8.1
  * Copyright Next Step Webs, Inc.
- * @link https://github.com/NextStepWebs/simplemde-markdown-editor
```

Comment:
# **_no_**

Reply:
Can you elaborate your objection? That repository currently linked to is empty and it should be updated.